pandas/core/window/numba_.py

```diff
@@ -141,7 +142,7 @@ def groupby_ewma(
 
                     if is_observation or not ignore_na:
 
-                        old_wt *= old_wt_factor
+                        old_wt *= old_wt_factor ** deltas[start + j - 1]
```

Comment:
why -1?

Reply:
We want to use ``delta[i-1]`` with ``values[i]``. ``cur = window[j] = values[start + j]`` (and ``j`` starts at 1).

I could introduce a new variable ``delta_window = deltas[start:stop-1]`` as in the cython implementation if you think that would make it clearer.

Reply:
no its fine, if you'd just add some documentation to this effect for future readers